src/SpotifyApi.NetCore/ILibraryApi.cs

```diff
@@ -77,28 +55,36 @@ public interface ILibraryApi
             string[] trackIds,
             string accessToken = null
             );
+        #endregion
 
         /// <summary>
-        /// Check if one or more tracks is already saved in the current Spotify user’s ‘Your Music’ library.
+        /// Check if one or more albums or shows or tracks is already saved in the current Spotify user’s ‘Your Music’ library.
         /// </summary>
-        /// <param name="trackIds">Required. A comma-separated list of the Spotify IDs for the tracks. Minimum: 1 ID. Maximum: 50 IDs.</param>
+        /// <param name="type">Required. Value is either album or show or track.</param>
+        /// <param name="albumOrShowOrTrackIds">Required. A comma-separated list of the Spotify IDs for the albums or shows or tracks. Minimum: 1 ID. Maximum: 50 IDs.</param>
+        /// <param name="accessToken">The bearer token which is gotten during the authentication/authorization process.</param>
         /// <returns>bool[] an array of true or false values, in the same order in which the ids were specified.</returns>
         /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/library/check-users-saved-albums/
+        /// https://developer.spotify.com/documentation/web-api/reference/library/check-users-saved-shows/
         /// https://developer.spotify.com/documentation/web-api/reference/library/check-users-saved-tracks/
         /// </remarks>
-        Task<T> CheckUsersSavedTracks<T>(
-            string[] trackIds,
+        Task<T> CheckUsersSavedAlbumsOrShowsOrTracks<T>(
+            string type,
+            string[] albumOrShowOrTrackIds,
```

Comment:
Don't put `CheckUsersSavedAlbumsOrShowsOrTracks` on the interface. This is should be `internal`. Leave the generic methods on the Interface